cyclic_boosting/base.py

```diff
@@ -633,6 +707,8 @@ def _fit_main(self, X: np.ndarray, y: np.ndarray, pred) -> np.ndarray:
             self.iteration_ += 1
 
         for feature in self.features:
+            #  TODO: It seems like this transformation could be done a method of the Feature class
```

Comment:
@fwick-panasonic , I was unsure if these steps inside the `for loop` could be refactored as a method of the `Feature` class. I think they can.

Reply:
Yes, I think that's ok.